components/compat-table/index-common.css

```diff
@@ -344,38 +346,21 @@ table {
     background: var(--color-border-primary);
   }
 
-  /* complicated selector to cover the last bit of the grey line above. */
-  .bc-notes-wrapper:last-child dd:last-child {
-    position: relative;
-
-    &::before {
-      position: absolute;
-      bottom: 0;
-      left: 7px;
-      z-index: -1;
-
-      width: 2px;
-      height: calc(100% - 6px);
-
-      content: "";
-
-      background: var(--color-background-primary);
-    }
-  }
```

Comment:
Could this removal have caused the grey line to continue beyond the last item?

To reproduce, expand the Chrome or Opera cell for `alert` in the `Window` BCD table:
https://fred-pr276.review.mdn.allizom.net/en-US/docs/Web/API/Window#browser_compatibility

Reply:
Should be fixed now